src/main/java/frc/robot/subsystems/pivot/PivotIOSim.java

```diff
@@ -56,7 +58,10 @@ public void setVoltage(double volts) {
   }
 
   @Override
-  public void setPivotAngle(double angle) {
-    pivotController.calculate(angle, pivotSim.getAngleRads());
+  public void setPivotAngle(double angleRots) {
+    double angleRads = angleRots * Math.PI * 2;
+    double currentPivotAngleRots = (pivotSim.getAngleRads() / (Math.PI * 2));
+    double armFF = armFeedforward.calculate(angleRads, pivotController.getSetpoint().velocity);
```

Comment:
keep units consistent to rotations. 
If you are calculating the pid setpoint in rotations you should do the feedforward in rotations, not radians.